Linebreak corrections (#373)
CloudCannon/platform-documentation: articles/configure-your-markdown-engine.mdx

  Ensure '/' is added to self-closing HTML tags (e.g. `<br />`) when saved back to your file. 
 </comp.DataReferenceRow>
 <comp.DataReferenceRow label="options.breaks" type_markdown="boolean">

>   You can use this option to match the Visual Editor's behaviour with the way your SSG's Markdown engine renders newline characters.

Reviewer: ```suggestion
  This key allows you to configure how the Visual Editor renders newline characters.
```

>   If `true`, newline characters within paragraphs will be rendered as `<br>` tags in the Visual Editor. CloudCannon will change all `<br>` tags in the file into newline characters before saving, to preserve your original formatting.

Reviewer: ```suggestion
  If `true`, CloudCannon will render newline characters within paragraphs as `<br>` tags in the Visual Editor. CloudCannon will also change all `<br>` tags in the file into newline characters before saving to preserve your original formatting.
```

>   If `false`, newline characters within paragraphs will be rendered normally when converted to HTML in the Visual Editor (i.e. collapsed by default). Editors can use Shift + Enter to add a linebreak within a paragraph. CloudCannon will save this as a `<br>` to ensure the linebreak appears when the site is built.

Reviewer: ```suggestion
  If `false`, CloudCannon will render newline characters within paragraphs normally when converted to HTML in the Visual Editor (i.e., collapsed by default). You can use Shift + Enter to add a linebreak within a paragraph. CloudCannon will save this as a `<br>` to ensure the linebreak appears when the site is built.
```
